### PR TITLE
[#139681] link to cross facility reports, fix export raw in cross facility

### PR DESCRIPTION
**Pull request by meara**
https://pm.tablexi.com/issues/139681

### CODE REVIEW

**Comment by jhanggi** [comment]
Let's add a couple tests around this. There are also circle failures and code climate changes to be made.

**Comment by meara** [comment]
@jhanggi updated

File: app/views/facilities/list.html.haml [hunk 1]
```diff
@@ -11,6 +11,10 @@
         %th= Facility.human_attribute_name(:name).titlecase
         %th= Facility.human_attribute_name(:status).titlecase
     %tbody
+      -if current_user.administrator?
```
Comment (jhanggi) [inline app/views/facilities/list.html.haml]: Should we feature flag this for now? Hopefully, eventually we can open it up at some point, but I'm not sure I want to give this to NU/UIC just yet without making sure it doesn't kill the servers.

Comment (meara) [reply app/views/facilities/list.html.haml]: What are we worrying about that would kill the servers? Happy to feature flag it, but not sure I understand exactly what the concern is.

Comment (jhanggi) [reply app/views/facilities/list.html.haml]: The products/accounts/etc tabs could eat up a huge chunk of memory and or time. I'd like to see what sort of timings we get from this once we're actually on a production machine. As long as we can still access the page via url hacking.

**Pull request merged**